versions.gradle

```diff
@@ -1,33 +1,15 @@
 ext {
-    // build dependencies
-    gradleBuildToolsVersion = '7.3.1'
-    gradleKotlinVersion = '1.7.20'
+    // build dependencies - not managed by refreshVersions
```

Comment:
Most dependency versions now managed in `versions.properties`

Reply:
only ones it couldn't move were the build plugins for dokka and gradle ktlint... might be able to fix that later.